packer/provision/local-builder.yaml

```diff
@@ -5,13 +5,16 @@
   become_method: sudo
 
   tasks:
```

Comment:
This entire section should be wrapped in a

when: ansible_distribution == 'RedHat'

Otherwise it would fire on Ubuntu nodes as well and fail

Reply:
Added as per suggestion. Please review.